hasher-matcher-actioner/scripts/hma_script_utils.py

```diff
@@ -269,51 +279,167 @@ def delete_action_rule(
         self.session.delete(self._get_request_url(api_path + action_rule_name))
 
 
-if __name__ == "__main__":
-    # If you want hard code tests for methods you can do so here:
+def get_terraform_outputs(
+    directory: str = "/workspaces/ThreatExchange/hasher-matcher-actioner/terraform",
+):
+    cmd = ["terraform"]
+    cmd.extend(["output", "-json"])
+    out = subprocess.check_output(cmd, cwd=directory)
+    return json.loads(out)
 
-    # Testing HasherMatcherActionerAPI
-    #   Since the API requries a deployed instance the majority of
-    #   testing needs to be done manually. See below.
 
-    # Add init values for the api:
-    # `os.environ.get(...)` will enable us pass values from `terraform output` in sh
+def get_terraform_outputs_from_file(
+    path: str = "/workspaces/ThreatExchange/hasher-matcher-actioner/tmp.out",
```

Comment:
Who would be populating this file?

Reply:
yeah, that path specifically was to test that it works. 
The idea was that someone would want to use the CLI somewhere besides where they deployed hma using tf (i.e. on an ec2 instance for example)